Sources/PackageLoading/Module+PkgConfig.swift

```diff
@@ -78,7 +86,23 @@ private extension SystemPackageProvider {
         }
         return false
     }
-    
+
+    func pkgConfigSearchPath() -> AbsolutePath? {
+        switch self {
+        case .Brew(let name):
+            // Homebrew can have multiple versions of the same package. The
+            // user can choose another version than the latest by running
+            // ``brew switch NAME VERSION``, so we shouldn't assume to link
+            // to the latest version. Instead use the version as symlinked
+            // in /usr/local/opt/(NAME)/lib/pkgconfig.
+            guard let brewPrefix = try? Utility.popen(["brew", "--prefix"]).chomp() else {
+                return nil
+            }
+            return AbsolutePath("\(brewPrefix)/opt/\(name)/lib/pkgconfig")
+        default: return nil
```

Comment:
Instead of default case, lets write all the cases so whenever a case is added we're forced to rethink about this method